pkgs/development/compilers/ghc/common-hadrian.nix

```
@@ -859,6 +859,12 @@ stdenv.mkDerivation (
       ghc-settings-edit "$settingsFile" \
       "windres command" "${toolPath "windres" installCC}"
     ''
+    + lib.optionalString stdenv.targetPlatform.isGhcjs ''
+      ghc-settings-edit "$settingsFile" \
+        "JavaScript CPP command" "${toolPath "cc" installCC}"
+      ghc-settings-edit "$settingsFile" \
+        "JavaScript CPP flags" "-E" 
```

```suggestion
        "JavaScript CPP flags" "-E"
```

treefmt rightly complains about trailing whitespace here